docs/index.rst

```diff
@@ -422,19 +429,24 @@ Displays messages logged by the current page.
 
 .. image:: logging.png
 
+.. _panel_performance:
+
 Performance
 ~~~~~~~~~~~
 
 Displays timing information, and, if enabled, Python profiling information
 for the current page.  When it is red, only timing will be done and
 no profiling information.
 
-.. note::
+There are two ways to enable the internal profiler used by the
+:guilabel:`Performance` panel.
 
-   An internal profiler can be enabled through the "performance" checkmark
-   in the "Settings" tab in the navigation bar. When the checkbox is green,
-   the request will be profiled and profiling information will be gathered and
-   displayed on the "Performance" panel output.
+#.  Under the :guilabel:`Settings` tab in the navigation bar, click the red :guilabel:`X` mark.
+    When there is a green :guilabel:`check` mark, each request will be profiled and profiling information will be gathered and displayed on the :guilabel:`Performance` panel output.
+    See :ref:`toolbar_settings_performance` [:guilabel:`Toolbar Settings`].
```

Comment:
This is misuse of [`:guilabel:`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-guilabel). It is used to distinguish GUI elements from inline text. There is no GUI element with the label of "Toolbar Settings".
```suggestion
    See :ref:`Toolbar Settings <toolbar_settings_performance>`.
```